content/05.main-text.md

```diff
@@ -386,6 +386,47 @@ TODO: somewhere link to notebook https://github.com/greenelab/hetmech/blob/04206
 
 ### Gamma-hurdle distribution
 
+We are interested in identifying source and target nodes whose connectivity exceeds what typically arises at random.
+To identify such especially-connected nodes, we compare DWPC values to the distribution of permuted network DWPC values for the same source and target nodes.
+While a single DWPC value is not actually a test statistic, we use a framework akin to classical hypothesis testing to identify outliers.
+
+Two observations led us to the quasi significance testing framework we developed.
+First, a sizable fraction of permuted DWPC values are often zero, indicating that the source and target nodes are not connected along the metapath in the permuted network.
+Second, we observed that non-zero DWPC values for any given source and target nodes are reasonably approximated as following a gamma distribution.
+Motivated by these observations, we parametrized permuted DWPC values using a zero-inflated gamma distribution, which we termed the gamma-hurdle distribution.
+We fit a gamma-hurdle distribution to each combination of source node, target node, and metapath.
+Finally, we estimate the probability of observing a permuted DWPC value greater than DWPC computed in the unpermuted network, akin to a one-tailed p-value.
+These quasi significance scores ('p-values') allow us to identify outlier node pairs at the metapath level.
+
+#### Details of the gamma-hurdle distribution
+
+Let $X$ be a gamma-hurdle random variable with parameters $\lambda$, $\alpha$, and $\beta$.
+
+$$X \sim \Gamma_H(\lambda, \alpha, \beta)$$
+
+The probability of a draw from the distribution is
+
+\begin{align}
+  P(X = 0) &= 1 - \lambda \\
+  P(X \in A; A \subseteq (0, \infty)) &= \frac{\lambda \beta^\alpha}{\Gamma(\alpha)}  \int  _{x \in A} \bigg( x^{\alpha - 1} e^{-\beta x} \bigg)
+\end{align}
+
+We estimate all three parameters using the method of moments (using Bessel's correction to estimate the second moment).
+As a validation of our method, we compared our method of moments parameter estimates to approximate maximum likelihood estimates (gamma distribution parameters do not have closed-form maximum likelihood estimates) and found excellent concordance between the methods.
```

Comment:
I just added some additional comments to https://github.com/greenelab/hetmech/pull/157. It might be good to merge that PR now so we can cite the repository itself (or an nbviewer) here.

Reply:
done in https://github.com/greenelab/hetmech/commit/025bdf8d5e63725ca2482d61fd8e421bf0001f93